backend/app/services/users/put_avatar.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Users
+  class PutAvatar < Core::Service
+    def call(url_params, current_user)
```

Comment:
The less you provide — the better. We're using only `signed_blob_id` from `url_params`. No need to pass the whole object then